fastexcel-writer/src/main/java/org/dhatim/fastexcel/Worksheet.java

```diff
@@ -855,9 +860,12 @@ private static void writeCol(Writer w, int columnIndex, double maxWidth, boolean
      * Helper method to get a cell name from (x, y) cell position.
      * e.g. "B3" from cell position (2, 1)
      */
-    private static String getCellMark(int row, int coll) {
-        char columnLetter = (char) ('A' + coll);
-        return String.valueOf(columnLetter) + String.valueOf(row+1);
+    static String getCellMark(int row, int coll) {
+        if (0 <= coll && coll <= 'Z' - 'A') {
```

Comment:
Why not use colToString always here? Perhaps make colToString static, so there's no need for the REF_HELPER